src/Go/boardState/goStyles.ts

```diff
@@ -1,9 +1,31 @@
 import { Theme } from "@mui/material/styles";
-import makeStyles from "@mui/styles/makeStyles";
-import createStyles from "@mui/styles/createStyles";
+import { makeStyles } from "tss-react/mui";
+import { keyframes } from "tss-react";
 
-export const pointStyle = makeStyles((theme: Theme) =>
-  createStyles({
+export type sizes = "fiveByFive" | "sevenBySeven" | "nineByNine" | "thirteenByThirteen" | "nineteenByNineteen";
```

Comment:
Types should be singular and capitalized (e.g. `"fiveByFive"` makes sense to be of type `Size`, not type `sizes`)

It looks like these style category types don't need to be exports since they are only used in this file